ProjectApplication/project_core/filters.py

```diff
@@ -6,22 +6,24 @@
 
 
 class ProjectFilterSet(FilterSet):
-    key_title = CharFilter(method='filter_key_title', label="Key or Title")
-    keywords = CharFilter(method='filter_keywords')
+    key_title = CharFilter(method='filter_any', label="Key, Title, Keyword, PI, laysummary or organisation")
     geographical_areas = CharFilter(method='filter_geographical_areas')
     funding_instrument = CharFilter(method='filter_funding_instrument')
     start_date = CharFilter(method='filter_start_date')
 
-    def filter_key_title(self, queryset, name, value):
+    def filter_any(self, queryset, name, value):
```

Comment:
It would be nice to do some word stemming (to find "penguin" if a user searches for "penguins") or proper full text search. Django only supports it for Postgresql, not for MariaDB (as far as I know!). And tokenization so the words don't need to be literally together if the users expect an "or" (e.g. to find "penguins" or "seals" if the user searches "penguins seals" (no quotes). Since Google does it I find that users expect it... ("or" by default).